Kickstarter-iOS/Views/Cells/SettingsTextInputCell.swift

```diff
@@ -0,0 +1,119 @@
+import Prelude
+import UIKit
+
+private struct Margin {
```

Comment:
In the future, we can extract this to common constants file, but for now we can keep it here